content/en/serverless/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Serverless
+kind: documentation
+aliases:
+  - /graphing/infrastructure/cloudfunctions
+  - /graphing/infrastructure/serverless_functions
+  - /graphing/infrastructure/serverless/
+  - /infrastructure/serverless/
+further_reading:
+- link: "/integrations/amazon_xray/"
+  tag: "X-Ray Integration"
+  text: "AWS X-Ray Integration"
+- link: "/integrations/amazon_lambda/"
+  tag: "AWS Lambda Integration"
+  text: "AWS Lambda Integration"
+---
+
+## Overview
+
+Serverless is a concept where you write event-driven code and upload it to a cloud provider, which manages all of the underlying compute resources. [Datadog Serverless][1] brings together metrics, traces, and logs from your AWS Lambda functions running serverless applications into one view.
+
+Make sure to check out discussions going on in the [#serverless][2] channel in the [Datadog Slack community][3].
+
+## Getting Started
+
+### 1. Install the AWS Integration
+
+Start by installing the [AWS Integration][4]. Once this is completed, view all of your Lambda Functions in the [Datadog Serverless Homepage][1].
```

Comment:
```suggestion
Start by installing the [AWS Integration][4]. This will allow Datadog to ingest CloudWatch metrics from AWS Lambda. The AWS integration installation also configures the Datadog Forwarder, which is required for ingestion of Lambda enhanced metrics, custom metrics and logs. 

Once the installation is complete, view all of your Lambda Functions in the [Datadog Serverless Homepage][1].
```